group.go

```diff
@@ -16,6 +16,7 @@ type Group struct {
 	DescendantServicesConnection     Connection `json:"-" graphql:"descendantServices"`
 	DescendantSubgroupsConnection    Connection `json:"-" graphql:"descendantSubgroups"`
 	DescendantTeamsConnection        Connection `json:"-" graphql:"descendantTeams"`
+	ChildTeamsConnection             Connection `json:"-" graphql:"childTeams"`
```

Comment:
With the way we wrote our sub queries - we no longer need this.  This is not used anywhere in the code it seems.  I suspect the other connection fields are also no longer used?

Reply:
Yeah, since it gets the page info from the query within the ChildTeams function?

Reply:
Removed all the *Connection fields